static/styles/zulip.scss

```diff
@@ -2728,7 +2728,7 @@ div.topic_edit_spinner .loading_indicator_spinner {
 }
 
 .alert-word {
-    background-color: hsla(102, 85%, 57%, 0.5);
+    background-color: rgba(181, 247, 153, 1.0);
```

Comment:
Can you explain why you've switched to rgba from hsla? We could just use hsl and remove the last `a` value for this.